src/App.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/home';
+import Navbar from './components/navigation';
+import Quotes from './components/quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/200790000-46ffb339-3546-42f7-8204-3e6cee2a13e8.png)

- [ ] Your readme file you don't have a descriptive title. Kindly change it to **Math Magician** to make your readme file more descriptive :blush: